pkgs/applications/misc/foot/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchgit, fcft, freetype, pixman, libxkbcommon, fontconfig, wayland
+, meson, ninja, ncurses, scdoc, tllist, wayland-protocols, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "foot";
+  version = "1.4.4";
+
+  src = fetchgit {
+    url = "https://codeberg.org/dnkl/foot.git";
+    rev = "${version}";
+    sha256 = "1cr4sz075v18clh8nlvgyxlbvfkhbsg0qrqgnclip5rwa24ry1lg";
+  };
+
+  nativeBuildInputs = [ meson ninja ncurses scdoc tllist wayland-protocols ];
+  buildInputs = [ fontconfig freetype pixman wayland libxkbcommon fcft pkg-config ];
+
+  # recommended build flags for foot as per INSTALL.md
+  preConfigure = ''
+    export CFLAGS+="-O3 -fno-plt"
+  '';
+
+  mesonFlags = [ "--buildtype=release" "-Db_lto=true" ];
```

Comment:
I *think Nix' meson setup hook sets this build release flags by default. Also, could try enabling tests? https://codeberg.org/dnkl/fcft/src/branch/master/meson.build#L99

Reply:
I checked and it doesn't seem like it is added (unless its a meson default behavior maybe):
https://github.com/NixOS/nixpkgs/blob/914b9788fd18a0eb908c7bf630cb206f5dabd25e/pkgs/development/tools/build-managers/meson/setup-hook.sh#L9-L17

Enabled the tests for fcft in 5bb0129062e05655d3408ebdc5e3a27141cefd83, good catch!